app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ActiveRecord::Base
+  validates :email, uniqueness: { case_sensitive: false }
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
```

Comment:
Sounds good.  I'm not familiar with all of them, but there might be use for something like ':trackable', but I think it's probably best to do with the minimum required.
I'm not sure that even 'remeberable' is useful, since it's an api, not an actual user -- so I don't think there is a browser to save the cookie in.